oremda/registry.py

```diff
@@ -94,8 +94,9 @@ def _inspect(self, image_name: str):
         image: Any = self.container_client.image(image_name)
         return image.labels
 
-    def _info(self, image_name: str):
-        return self.images.get(image_name)
+    def _info(self, image_name: str) -> ImageInfo:
+        info: Any = self.images.get(image_name)
+        return info
```

Comment:
This is really a hotfix for an issue introduced in your last PR, where the return type of `_info()` became `ImageInfo | None` due to the fact that some logic was moved to `register()`. However, several methods in that class expect `_info()` to strictly return `ImageInfo`, and that caused a number of typing errors.

I'll change the body to `return self.images[image_name]`, and let a `KeyError` be raised if the image wasn't already inspected

Reply:
Ah I see. Thanks, and sorry about that change in behavior. I think allowing a `KeyError` is a good way to go.

Reply:
I'm still not totally convinced we should be modifying code in order to improve experiences with editors. But I'll be thinking about it.

Reply:
I would argue that adding type is not just about improving experience with editors, through auto complete. They can also catch certain errors ( if you run static type checker, which is what some editors do ) and help to document the code/improve readability.

Reply:
@cjh1 Yes that makes sense. I wasn't specifically commenting on typing in general, though, but this specific case of adding a variable `info` and giving it the type `Any`. I think this is fine for hot fixing. And I guess this could be a reasonable hot fix for getting the static type checker to pass. But I wouldn't think this improves readability.

Reply:
So it seems that we need to go through the callers of `_info` and correct the type expectation?